app/javascript/packages/document-capture/components/in-person-location-full-address-entry-post-office-search-step.spec.tsx

```diff
@@ -36,10 +36,12 @@ const USPS_RESPONSE = [
   },
 ];
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const defaultRegisterField = (string) => undefined;
 const DEFAULT_PROPS = {
   toPreviousStep() {},
   onChange() {},
-  registerField() {},
+  registerField: defaultRegisterField,
```

Comment:
Could we instead update `registerField` to support `void` as a supported return type?

https://github.com/18F/identity-idp/blob/a5c23dcb990ff60d1b04d4d3631116c5f784c805/app/javascript/packages/form-steps/form-steps.tsx#L35

➡️ 

```tsx
) => undefined | void | RefCallback<HTMLElement>;
```

Reply:
I removed the inline exceptions, but adding `void` didn't seem to work.

Unfortunately adding `void` as a return type causes more type issues than it solves. An example:
```javascript
// `app/javascript/packages/address-search/components/address-input.tsx`
        <TextInput
          required
          ref={registerField('address')} // This is either undefined or `ForwardedRef<HTMLInputElement>` adding 'void' here seems like a bad idea.
```

Reply:
Oof. Okay, in that case, maybe we could at least inline the default?

```tsx
  registerField: () => undefined,
```

Reply:
Much [Inline the two default functions](https://github.com/18F/identity-idp/pull/10521/commits/20c0596dc8aa2db5dc5c7ee185eab040389fd243). For some reason I couldn't get the inline function to work before, but now it works. 👍